functions/index.md

```diff
@@ -0,0 +1,11 @@
+# Get started with PostgreSQL functions
+TimescaleDB is built on top of PostgreSQL. When you interact with TimescaleDB,
+you're usually using PostgreSQL functions. This guide helps you get up and
+running with the most common PostgreSQL functions that you need to use wirth
+TimescaleDB.
+
+* [`ORDER BY`][order-by] to order the results of your query
+* [`INSERT`][insert] to insert data into a hypertable
+
+[order-by]: /functions/order-by
+[insert]: /functions/FIXME
```

Comment:
If this is a top-level section, I believe these would be in the style `/functions/:currentVersion:/FIXME`